dbt_project.yml

```diff
@@ -5,7 +5,10 @@ version: '0.1.0'
 require-dbt-version: [">=1.0.0", "<2.0.0"]
 
 # This setting configures which "profile" dbt uses for this project.
-profile: 'dbt_synth_data'
+# profile: 'snowflake-1'
+profile: 'redshift-1'
+# profile: 'bigquery-1'
+# profile: databricks-1
```

Comment:
DBT YAML's profile shouldn't change. Maybe, you need to change target.

Reply:
now we have profile: 'benchmark' and use target